examples/bri-1/base-example/test/e2e.spec.ts

```diff
@@ -198,13 +198,15 @@ describe('baseline', () => {
 
           describe(`invited workgroup organization: "${aliceCorpName}"`, shouldBehaveLikeAnInvitedWorkgroupOrganization.bind(this));
           describe(`workgroup organization: "${aliceCorpName}"`, shouldBehaveLikeAWorkgroupOrganization.bind(this));
-          describe(`workgroup counterparty: "${aliceCorpName}"`, shouldBehaveLikeAWorkgroupCounterpartyOrganization.bind(this));
         });
 
-        describe("Bob's organization after Alice has joined", function () {
-          before(async () => { this.ctx.app = bobApp; });
+        describe('counterparties', function () {
+          before(async () => {
+            this.ctx.app = bobApp;
```

Comment:
This line sets app that will be tested by `shouldBehaveLikeAWorkgroupCounterpartyOrganization`. The way these tests are constructed means `bobApp` will get tested twice instead of testing `bobApp` once and `aliceApp` once. The `describe(aliceCorpName, shouldBehaveLikeAWorkgroupCounterpartyOrganization.bind(this));` line should be moved to a different `describe` block where `this.ctx.app` is set to `aliceApp`

Reply:
We fixed this.